config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml

```diff
@@ -125,15 +125,6 @@ spec:
          ctlplane_dns_nameservers:
          - 192.168.122.1
          dns_search_domains: []
-         registry_url: quay.io/podified-antelope-centos9
-         image_tag: current-podified
-         image_prefix: openstack
-         edpm_ovn_controller_agent_image: "{{ registry_url }}/{{ image_prefix }}-ovn-controller:{{ image_tag }}"
-         edpm_iscsid_image: "{{ registry_url }}/{{ image_prefix }}-iscsid:{{ image_tag }}"
-         edpm_logrotate_crond_image: "{{ registry_url }}/{{ image_prefix }}-cron:{{ image_tag }}"
-         edpm_nova_compute_container_image: "{{ registry_url }}/{{ image_prefix }}-nova-compute:{{ image_tag }}"
-         edpm_nova_libvirt_container_image: "{{ registry_url }}/{{ image_prefix }}-nova-libvirt:{{ image_tag }}"
-         edpm_neutron_metadata_agent_image: "{{ registry_url }}/{{ image_prefix }}-neutron-metadata-agent-ovn:{{ image_tag }}"
```

Comment:
Yeah, I was wondering if all of these had a practical purpose. Good to drop them.

I thought maybe the ci-framework was using kustomize to replace the URL's for internal, downstream testing.